Entities/Industry/Drill/Drill.as

```diff
@@ -273,7 +279,7 @@ void onTick(CBlob@ this)
 								// or blobs to increase damage to (for the future)
 								string name = b.getName();
 
-								if (name == "mat_stone" || name == "mat_wood" || name == "mat_gold")
+								if (b.hasTag("material") || b.hasTag("invincible") || b.hasScript("Eatable.as") || name == "tent")
```

Comment:
All materials, food, and the tent are given the "invincible" tag in IgnoreDamage.as (except for a few exceptions for items that are rarely/never used), so having the extra checks for tags is redundant.